tests/localosmosis/README.md

```diff
@@ -77,134 +77,39 @@ make localnet-clean
 
 ## 2. LocalOsmosis - With Mainnet State
 
-Running LocalOsmosis with mainnet state is resource intensive and can take a bit of time.
-It is recommended to only use this method if you are testing a new feature that must be thoroughly tested before pushing to production.
+Running an osmosis network with mainnet state is now as easy as setting up a stateless localnet.
 
-A few things to note before getting started. The below method will only work if you are using the same version as mainnet. In other words,
-if mainnet is on v8.0.0 and you try to do this on a v9.0.0 tag or on main, you will run into an error when initializing the genesis.
-(yes, it is possible to create a state exported testnet on a upcoming release, but that is out of the scope of this tutorial)
+1. Set up a mainnet node and stop it at whatever height you want to fork the network at.
 
-Additionally, this process requires 64GB of RAM. If you do not have 64GB of RAM, you will get an OOM error.
+2. There are now two options you can choose from:
 
-### Create a mainnet state export
+   - **Mainnet is on version X, and you want to create a testnet on version X.**
 
-1. Set up a node on mainnet (easiest to use the [get.osmosis.zone](https://get.osmosis.zone) tool). This will be the node you use to run the state exported testnet, so ensure it has at least 64GB of RAM.
+     On version X, run:
 
-```sh
-curl -sL https://get.osmosis.zone/install > i.py && python3 i.py
-```
-
-2. Once the installer is done, ensure your node is hitting blocks.
-
-```sh
-source ~/.profile
-journalctl -u osmosisd.service -f
-```
-
-3. Stop your Osmosis daemon
-
-```sh
-systemctl stop osmosisd.service
-```
-
-4. Take a state export snapshot with the following command:
-
-```sh
-cd $HOME
-osmosisd export 2> state_export.json
-```
-
-After a while (~15 minutes), this will create a file called `state_export.json` which is a snapshot of the current mainnet state.
-
-### Use the state export in LocalOsmosis
-
-1. Copy the `state_export.json` to the `LocalOsmosis/state_export` folder within the osmosis repo
+      ```bash
```

Comment:
can we add a description on what the second args of acc address does here?

Reply:
https://github.com/osmosis-labs/osmosis/pull/7819/commits/b1b8aa7796c46fbd663b8428794be45d71323bc0